src/collections/_documentation/platforms/javascript/react.md

```diff
@@ -35,34 +35,39 @@ In development mode React will rethrow errors caught within an error boundary. T
 %}
 
 ```jsx
-import React from 'react';
+import React, { Component } from 'react';
 import * as Sentry from '@sentry/browser';
 
 class ExampleBoundary extends Component {
     constructor(props) {
         super(props);
-        this.state = { error: null, eventId: null };
+        this.state = { eventId: null };
     }
 
-    componentDidCatch(error, errorInfo) {
-      this.setState({ error });
-      Sentry.withScope(scope => {
-          scope.setExtras(errorInfo);
-          const eventId = Sentry.captureException(error);
-          this.setState({eventId})
-      });
-    }
+  static getDerivedStateFromError(error) {
+    return { hasError: true };
+  }
+
+  componentDidCatch(error, errorInfo) {
+    Sentry.withScope(scope => {
+      scope.setExtra(errorInfo);
+      const eventId = Sentry.captureException(error);
+      this.setState({ eventId });
+    });
+  }
```

Comment:
Indentation here looks inconsistent with other parts of the code in the same class, could we get that straightened out?

Reply:
@markstory yes indeed, I fixed it